client/web/src/styles/design-tokens.json

```diff
@@ -0,0 +1,46 @@
+{
+   "design_tokens": {
+      "color": {
+         "accent": {
+            "$value": "#6F5F11",
+            "$type": "color"
+         },
+         "neutral": {
+            "$value": "#001561",
+            "$type": "color"
+         }
+      },
+      "fontFamily": {
+         "bodyFont": {
+            "$value": "Verdana",
```

Comment:
I'll update the editor to include the entire css string. In design-system-editor we have set default bodyFont to be '"Segoe UI", Arial, Helvetica, sans-serif' which is what it will pick up from in the blank app seed as well. Should I change that setting if we want the default to be Roboto?

Reply:
As discussed, we want the expanded CSS strings both here and in the editor